### PR TITLE
add ShowMeYourHotKeys 1.7

### PR DESCRIPTION
**Pull request by franckbrun**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [X] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [X] `brew audit --cask --online <cask>` is error-free.
```
➜  ~ brew audit --cask --online showmeyourhotkeys
==> Downloading https://showmeyourhotkeys.moxadventu.com/app/ShowMeYourHotKeys.zip
Already downloaded: /Users/franckb/Library/Caches/Homebrew/downloads/51ba2d3c3f988f20a5ac50dd16ea5271951fd3dd82946348c712c0733c835580--ShowMeYourHotKeys.zip
Warning: No checksum defined for cask 'showmeyourhotkeys', skipping verification.
```
- [X] `brew style --fix <cask>` reports no offenses.
```
➜  ~ brew style --fix showmeyourhotkeys

1 file inspected, no offenses detected

```

Additionally, **if adding a new cask**:

- [ X] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
```
➜  ~ $(brew --repository homebrew/cask)/developer/bin/generate_cask_token "ShowMeYourHotKeys"
Proposed token:               showmeyourhotkeys
Proposed file name:           showmeyourhotkeys.rb
Cask Header Line:             cask "showmeyourhotkeys" do
```
- [X] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [X] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [X] `brew audit --new-cask <cask>` worked successfully.
```
➜  ~ brew audit --new-cask showmeyourhotkeys
==> Downloading https://showmeyourhotkeys.moxadventu.com/app/ShowMeYourHotKeys.zip
Already downloaded: /Users/franckb/Library/Caches/Homebrew/downloads/51ba2d3c3f988f20a5ac50dd16ea5271951fd3dd82946348c712c0733c835580--ShowMeYourHotKeys.zip
Warning: No checksum defined for cask 'showmeyourhotkeys', skipping verification.
==> Downloading https://showmeyourhotkeys.moxadventu.com/app/ShowMeYourHotKeys.zip
Already downloaded: /Users/franckb/Library/Caches/Homebrew/downloads/51ba2d3c3f988f20a5ac50dd16ea5271951fd3dd82946348c712c0733c835580--ShowMeYourHotKeys.zip
Warning: No checksum defined for cask 'showmeyourhotkeys', skipping verification.
```
- [ ] `brew install --cask <cask>` worked successfully.
```
➜  ~ brew install showmeyourhotkeys
==> Downloading https://showmeyourhotkeys.moxadventu.com/app/ShowMeYourHotKeys.zip
Already downloaded: /Users/franckb/Library/Caches/Homebrew/downloads/51ba2d3c3f988f20a5ac50dd16ea5271951fd3dd82946348c712c0733c835580--ShowMeYourHotKeys.zip
Warning: No checksum defined for cask 'showmeyourhotkeys', skipping verification.
==> Installing Cask showmeyourhotkeys
==> Moving App 'ShowMeYourHotKeys.app' to '/Applications/ShowMeYourHotKeys.app'
🍺  showmeyourhotkeys was successfully installed!
```
- [ ] `brew uninstall --cask <cask>` worked successfully.
```
➜  ~ brew uninstall showmeyourhotkeys
==> Uninstalling Cask showmeyourhotkeys
==> Backing App 'ShowMeYourHotKeys.app' up to '/usr/local/Caskroom/showmeyourhotkeys/1.7/ShowMeYourHotKeys.app'
==> Removing App '/Applications/ShowMeYourHotKeys.app'
==> Purging files for version 1.7 of Cask showmeyourhotkeys
```

### CODE REVIEW

File: Casks/showmeyourhotkeys.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+cask "showmeyourhotkeys" do
+  version "1.7"
+  sha256 :no_check
+
+  url "https://showmeyourhotkeys.moxadventu.com/app/ShowMeYourHotKeys.zip"
+  name "showmeyourhotkeys"
```
Comment (bevanjkay) [inline Casks/showmeyourhotkeys.rb:6]:
```suggestion
  name "ShowMeYourHotKeys"
```

File: Casks/showmeyourhotkeys.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
+cask "showmeyourhotkeys" do
+  version "1.7"
+  sha256 :no_check
+
+  url "https://showmeyourhotkeys.moxadventu.com/app/ShowMeYourHotKeys.zip"
```
Comment (bevanjkay) [inline Casks/showmeyourhotkeys.rb:5]:
```suggestion
  url "https://showmeyourhotkeys.moxadventu.com/update/ShowMeYourHotKeys-prod-stable-#{version.csv.first}-(#{version.csv.second).zip"
```

File: Casks/showmeyourhotkeys.rb [hunk 3]
```diff
@@ -0,0 +1,18 @@
+cask "showmeyourhotkeys" do
+  version "1.7"
```
Comment (bevanjkay) [inline Casks/showmeyourhotkeys.rb:2]:
```suggestion
  version "1.7,20"
```

File: Casks/showmeyourhotkeys.rb [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+  sha256 :no_check
+
+  url "https://showmeyourhotkeys.moxadventu.com/app/ShowMeYourHotKeys.zip"
+  name "showmeyourhotkeys"
+  desc "Show applications menu items hotkeys"
+  homepage "https://showmeyourhotkeys.moxadventu.com/"
+
+  livecheck do
+    url "https://showmeyourhotkeys.moxadventu.com/update/appcast.xml"
+    strategy :sparkle, &:short_version
```
Comment (bevanjkay) [inline Casks/showmeyourhotkeys.rb:12]:
```suggestion
    strategy :sparkle
```

File: Casks/showmeyourhotkeys.rb [hunk 5]
```diff
@@ -0,0 +1,18 @@
+cask "showmeyourhotkeys" do
+  version "1.7"
+  sha256 :no_check
```
Comment (bevanjkay) [inline Casks/showmeyourhotkeys.rb:3]:
```suggestion
  sha256 "4cca26821cd0d715f1b90ea34e886b2814361b932ac23a502b31e418a6d55c4e"
```

**Review by bevanjkay — CHANGES_REQUESTED** [review]
Thanks @franckbrun for your submission to `hombrew-cask`. Please see my review comments and additionally - please try to add a [zap stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap). The linked documentation provides some recommended methods to create it.

**Comment by bevanjkay** [comment]
@franckbrun Sorry one of my suggestions had a typo! I've fixed it up. If you can just have a look into the zap stanza then I think we should be tracking along pretty close.

**Comment by franckbrun** [comment]
Hi @bevanjkay. Thank you for your review. I applied your suggestions and I added a zap stanza. Currently testing those changes.

**Comment by bevanjkay** [comment]
@franckbrun Looks great! Thanks for your first contribution to `homebrew-cask` 🍻️

**Review by bevanjkay — APPROVED** [review]

**Pull request merged**